client/packages/invoices/src/Returns/modals/InboundReturn/useDraftInboundReturnLines.ts

```diff
@@ -25,5 +46,26 @@ export const useDraftInboundReturnLines = (stockLineIds: string[]) => {
     });
   };
 
-  return { lines: draftLines, update };
+  const saveInboundReturn = async () => {
+    const inboundReturnLines: InboundReturnLineInput[] = draftLines.map(
```

Comment:
I would usually do below, but this is completely stylistic preference, please don't change, just an option for future if you agree:

```TS
const inboundReturnLines: InboundReturnLineInput[] = draftLines.map(
      ({ id, reasonId, numberOfPacksReturned, stockLineId, comment }) => ({
        id,
        stockLineId,
        reasonId,
        comment,
        numberOfPacksReturned,
      })
    );
```

Reply:
ooo yes so much better love it